arch/ARM/cortex_m/src/nvic_cm4_cm7/cortex_m-nvic.ads

```diff
@@ -44,10 +44,13 @@
 
 with System;
 with HAL;            use HAL;
-with Ada.Interrupts; use Ada.Interrupts;
+-- with Ada.Interrupts; use Ada.Interrupts;
 
 package Cortex_M.NVIC is  -- the Nested Vectored Interrupt Controller
 
+   subtype Interrupt_ID is Natural range 0 .. 127;
```

Comment:
What are you worried about?

Reply:
I actually had to do a similar change:

```ada
with System;
with HAL;    use HAL;

package Cortex_M.NVIC is  -- the Nested Vectored Interrupt Controller

   type Interrupt_ID is range 0 .. 240;
```
I took 0 .. 240 because it is the max number of interrupts on a CM7.

Reply:
Mostly I didn't know if this change would have wider consequences for other targets. CI failed, but perhaps is was due to a cm7 target and my initial choice of a too small range.

Reply:
Seems like I could rebase with this change you've made and be good to go.

Reply:
Hrm. It looks like your change above hasn't landed in master. I'll add it to my branch and see if it passes CI.

Reply:
Yes, I forgot to add it to master. I can try to find the time for a dedicated pull-request.